src/glue/jobs/s3_to_json_s3.py

```diff
@@ -196,8 +200,8 @@ def get_json_schema(archive_map, file_metadata, json_schemas, self_ref_schema_li
             * url (str)
             * schema (dict)
             * app_id (str)
-            * assessment_id (int)
-            * assessment_revision (int)
+            * assessment_id (str)
+            * assessment_revision (str)
```

Comment:
These type changes were both corrections and make `file_metadata` more consistent across the functions.